components/public-api-server/pkg/proxy/prometheusmetrics_test.go

```diff
@@ -36,7 +34,7 @@ func TestConnectionCreationWasTracked(t *testing.T) {
 		WorkspaceId: "some-ID",
 	})
 
-	count, err := testutil.GatherAndCount(registry)
+	count, err := testutil.GatherAndCount(srv.MetricsRegistry())
 	assert.NoError(t, err)
 	assert.Equal(t, 1, count)
```

Comment:
This now fails because it found more. Is it because now we're using the global registry, there are more metrics that went along with it?

Reply:
That's because the `srv.MetricsRegistry()` has more metrics than just the 1. We can pass an argument to `GatherAndCount` to filter on only the ones we are testing. Will fix.